src/scss/main.scss

```diff
@@ -590,15 +583,14 @@ $loader-scale-duration: 300ms;
   border-color: $border-color;
   border-style: solid;
   border-width: 0 1px 1px 1px;
-  box-shadow: none;
   cursor: pointer;
   display: flex;
   justify-content: space-between;
   margin: 0 auto;
   padding: 14px 10px;
   position: relative;
-  transition: opacity 300ms, box-shadow 300ms, width 200ms $curve-easeOutBack, border-color 300ms, border-radius 300ms;
-  width: 99%;
+  transition: opacity 300ms, width 200ms $curve-easeOutBack, border-color 300ms, border-radius 300ms;
+  width: 98%;
```

Comment:
I think it'll just make the methods a bit smaller unless it's the active method.